.gitignore

```diff
@@ -22,4 +22,7 @@ yarn-error.log*
 
 /public/apps
 /public/aragon-ui
+
+# ignore lock files
 package-lock.json
+yarn.lock
```

Comment:
Hmm, not sure we should still do this; I think these lock files are actually useful since we're shipping this now.

The best case scenario is if we could only have it be added on the commits that tag a release, e.g. 0.5.

Reply:
The idea was to only release them in the npm package, but we could also have them again on the repo, and only update them when a new version is tagged. That would imply taking care of never updating these files outside of releases, and recommending to contributors to remove these files before `npm install`ing since they are not maintained between versions.

Another solution could also be to add it in the release commits, and remove them in the next commit (in that case we could even leave them in the .gitignore, and use `git add -f` to add them for release commits).

What do you think?

Reply:
This is such a headache D:

> Another solution could also be to add it in the release commits, and remove them in the next commit

This is sounding the most plausible for me... I think it'd be nice to have them in the commit of the release tag to make it easier to reproduce a certain npm version locally.

We could have a small release script that does this, similar to what you've already got in react-scripts :)